### ISSUE TITLE
my wallet has been changed over update

### ISSUE THREAD

**Issue by kk7nc** [helium/hotspot-app#1002]
in the past week, my app wash crashed and I did a screenshot from all of my 12 words but after logging in with that 12 words all my HNT and hotspots are gone,
I email several times to Helium support but they said that your current wallet is different from the wallet that is associated with a hotspot you used which is not make sense.  

and they only keep saying your 12 words is not the words you used for that wallet, I don't know what happened in this app,
Now they only say we cannot recover those words, but still, I don't know what happen with my hotspot.

This is not my fault to forget those words because I have them but I find the bug which if you log in in two devices like iPhone and android with the same words the wallet could be changed and everything will be gone.

now how can I transfer my hotspot based on what is happened to me, I just turn it off.

- Device:  iPhone and android
- OS: IOS 15 and android 11

**Comment by hotfinal** [comment]
I also got the same problem as kk7nc.
I have register Helium wallet about 2 weeks ago and added 1 Helium miner to the wallet.
I only create 1 wallet and write down the 12-words on the paper.
After the app failure, my balance shows $0 then I try to sign out and sign back in with my 12 words but the wallet still show $0. 
After I tried to find the solution on the internet and found the solution which is to change API from Stakejoy to Helium but it didn't work for me. 
lately I found out that my wallet address mysteriously changed to different one just the same situation as kk7nc.
I try to enter that 12-words to my another phone, it show the same new wallet.
I don't know why because I only have 1 set of 12-words
